### DISCUSSION TITLE
MQTTX blocks after Connect on windows

### DISCUSSION THREAD

**Discussion by Sikulo** [emqx/MQTTX#802] [Q&A]
Good day community
I have an issue that maybe can be solved quickly:

I installed MQTTX for windows and i set the connections: one remote and one local (localhost). 
As soon as i click Connect then i can't execute any other command, the icons and buttons and even texts seems blocked/not working/not usable and i have to close and restart the whole application...
Also the green button Connect doesn't work... If i fold the headers and i push the black arrow for connect it works.
But more problematic is the fact that i can't execute any command after Connect in localhost. 
Things works fine if i connect to remote hosts...

Thanks a lot )

**Comment by ysfscream** [comment]
Hi, sorry for my later reply! Does this seem a little strange? Are there a lot of messages from subscribed topics? Or did you download and try using https://github.com/emqx/MQTTX/releases/tag/v1.7.0

**Comment by Sikulo** [comment]
I just installed from the exe for windows and it seems a problem with localhost... ((

**Reply by ysfscream** [reply]
What's the problem?
